src/main/java/duke/exception/DukeInvalidArgumentException.java

```diff
@@ -0,0 +1,17 @@
+package duke.exception;
+
+/**
+ * Represents a duke exception when an invalid argument is provided for the command.
+ */
+public class DukeInvalidArgumentException extends DukeException {
```

Comment:
I like that you created subtypes for each common Exception type for greater readability!